atd-vzd/triggers/fatality_insert.sql

```diff
@@ -0,0 +1,29 @@
+-- This function inserts a new record into the fatality table
+-- It is triggered by fatal insertions to the primaryperson or person tables
+CREATE OR REPLACE FUNCTION fatality_insert() 
+    RETURNS trigger
+    LANGUAGE plpgsql
+AS $function$
+    DECLARE
+    	year integer;
+        location text;
+    BEGIN
+    	SELECT TO_CHAR(crash_date,'yyyy')
+            FROM atd_txdot_crashes
+            WHERE crash_id = NEW.crash_id
+            INTO year;
```

Comment:
Since this year is created from `crash_date` in the crashes table, I'm curious if this could be handled in the front end to keep us from storing the year info twice in the DB. Not sure what our options are in the React code so disregard if this is the best way. 👍

The location field makes sense to me since that is more complex.

Reply:
I'm thinking about this some more now that I submitted review, but should we think about how crash records edits could cause the data in this table to drift from the source? Thinking about `year` and `location` specifically.